src/peripheral/dcb.rs

```diff
@@ -22,10 +24,7 @@ pub struct RegisterBlock {
 }
 
 impl DCB {
-    /// Enables TRACE. This is for example required by the
-    /// `peripheral::DWT` cycle counter to work properly.
-    /// As by STM documentation, this flag is not reset on
-    /// soft-reset, only on power reset.
+    /// Global enable for all [`DWT`] and [`ITM`] features.
```

Comment:
I think just doing a doc comment would be better than importing:
```suggestion
    /// Global enable for all [`DWT`](crate::peripheral::DWT) and [`ITM`](crate::peripheral::ITM) features.
```